server/src/pages/shot/controller.js

```diff
@@ -43,10 +43,12 @@ exports.launch = function(data) {
     if (window.wantsauth.getAuthData()) {
       Object.assign(model, window.wantsauth.getAuthData());
       model.isExtInstalled = true;
+      model.isFxaAuthenticated = model.accountId && model.accountId === model.shotAccountId;
     } else {
       window.wantsauth.addAuthDataListener((data) => {
         Object.assign(model, data);
         model.isExtInstalled = true;
+        model.isFxaAuthenticated = model.accountId && model.accountId === model.shotAccountId;
```

Comment:
These lines are nearly identical to those in the `if`.  Consider putting them into a function.